public/scripts/data-util.js

```diff
@@ -1,17 +1,19 @@
 var DataUtil = (function() {
-    var data = {};
-
-    var cluster_name_mapping = null;
+  var data = {};
+
+  var cluster_name_mapping = [];
+  var job_name_prefix = '';
+
+  $.ajax({
+    async: false,
```

Comment:
Fine for now, but we should see about putting the config into localstorage or something so we don't have to do a blocking call to get it.